frontend/src/metadata.config.tsx

```diff
@@ -28,3 +28,32 @@ export const widgetTypes ={
     "DateTimePicker": "Date Time Picker",
     "Select": "Dropdown"
 }
+
```

Comment:
Created https://github.com/clowder-framework/clowder2/issues/255. Hope it's not a duplicate.